### PR TITLE
Feature: Video range support

### PR DESCRIPTION
**Pull request by RadinGospodinov**
# Playing only a portion of the video (start & end time)
Provide an optional `startTime` and/or `endTime` for the video. Value is in milliseconds. Useful when you want to play only a portion of a large video.
Example
```
source={{ startTime: 36012, endTime: 48500 }}
```

### CODE REVIEW

File: Video.js [hunk 1]
```diff
@@ -342,6 +342,8 @@ export default class Video extends Component {
         mainVer: source.mainVer || 0,
         patchVer: source.patchVer || 0,
         requestHeaders: source.headers ? this.stringsOnlyObject(source.headers) : {},
+        startTime: source.startTime,
```
Comment (freeboub) [inline Video.js]: maybe add || 0, then you ensure implementations get a correct value ?

**Comment by freeboub** [comment]
It is a very good PR, it you good point that you put new fields in src prop !
Can you please update API.md to add the new API ?
And also add a line in changeLog.
I will merge it rapidly ! 
Thank you

**Comment by yavor87** [comment]
Hey @freeboub , 
Sorry, we issued this PR a bit sooner than we intended by mistake. I have updated the api & changelog.

File: Video.js (hunk 1 above)
Comment (freeboub) [reply Video.js]: (Sorry, this is a long time comment I forgot to submit ...)

Comment (yavor87) [reply Video.js]: Thank you @freeboub .

**Comment by tslater** [comment]
@RadinGospodinov Does this normalize the other times of the video so that time 36012 on the video file is now time 0 (from your example? or is time 36012 still 36012?

Example

`source={{ startTime: 36012, endTime: 48500 }}`

**Comment by RadinGospodinov** [comment]
Correct, it creates ClippingMediaSource and 36012 will become 0. The duration of the clip will be 48500 - 36012, so if you need to seek will need to pass a in the range between 0 and 48500 - 36012

**Comment by tslater** [comment]
@RadinGospodinov  Thanks for the quick reply. I noticed that normally the `seek()` function takes a parameter in seconds, but the `startTime` and `endTime` are in milliseconds. 

Does using `startTime` and `endTime` in the source also change the seek time to seconds? (Seems like it might be the case from your example here?) 

If that's the case, would it also affect the `onProgress` event values as well?

**Comment by RadinGospodinov** [comment]
There are no changes to seek and in progress, so they should continue working the same way as without start and end time.

**Pull request merged**